OWML.ModLoader/ModLoader.cs

```diff
@@ -32,7 +34,8 @@ public static void LoadMods()
             var harmonyHelper = new HarmonyHelper(logger, console);
             var events = new ModEvents(logger, console, harmonyHelper);
             var menus = new ModMenus(logger, console, events);
-            var owo = new Owo(modFinder, logger, console, owmlConfig, menus, harmonyHelper);
+            var inputHandler = new ModInputHandler(logger, console, harmonyHelper);
```

Comment:
Do you want each mod to have a separate InputHandler, or a shared one? Right now it's shared. It's not wrong, just wanna make sure you're aware of the difference.

Reply:
shared ofc